classes.py

```diff
@@ -1382,13 +1386,19 @@ def collectItem(self, item_pos, level):
 
         item = level.getNode(x, y)
         
+        itemIsEquipment = False
         #if the item is an equipment, add it to the inventory
         if isinstance(item, Equipment):
+            itemIsEquipment = True
             self.inventory[item.getId()] = 1
             
         #increment score
         self.score += item.getScore()
         
+        event_type = "DaveCollectedItem" 
```

Comment:
Note that I'm probably going to rename the event_type to "CollectedItem" just to be consistent with the other custom events, that don't specifically have "Dave" in the name.

Reply:
makes sense